src/views/core/bodies/Edit.vue

```diff
@@ -69,7 +69,7 @@
         <div class="field">
           <label class="label">Body code <span class="has-text-danger">*</span></label>
           <div class="control">
-            <input class="input" type="text" required :disabled="!can.editCode" v-model="body.code" />
+            <input class="input" type="text" required :disabled="!can.editCode" v-model="body.code" :maxlength="3" :minlength="3"/>
```

Comment:
Can we use an input pattern for this? Like; https://www.w3schools.com/tags/att_input_pattern.asp